libs/design-system/src/pagination/ControlBar.tsx

```diff
@@ -1,5 +1,5 @@
 import styled from '@emotion/styled';
-import { Box, BoxProps } from '@mantine/core';
+import { Box, BoxProps, useMantineTheme } from '@mantine/core';
 import { forwardRef, PropsWithChildren, useContext, useEffect, useState } from 'react';
 import { ChevronLeft, ChevronRight } from '../icons';
 import { ControlButton } from './ControlButton';
```

Comment:
In the `ControlButton` I see that we need to override the `background-image` for the nested `label` component, because for me (and you, check the screenshot below) it doesn't look correct. With this code would be easier I guess:

```
.mantine-Button-label {
    background-image: none;
  }
```

I also see that something is wrong with the border of `Go to` component, but couldn't understand and find the reason.

![image](https://github.com/novuhq/novu/assets/2607232/336da2b6-26bc-4a06-9cd0-3ce8bdce5cae)

Reply:
Fixed the border, but sent you a message to better understand what you mean about the button label not looking visually correct!